### PR TITLE
Adding Azure Documentation

### PR DESCRIPTION
**Pull request by danigian**
Signed-off-by: Daniele Antonio Maggio <dcharizard@gmail.com>

### CODE REVIEW

**Comment by danigian** [comment]
@alexellis please check this new PR. Don't know what happened before

File: community.md [hunk 1]
```diff
@@ -18,8 +18,6 @@ There are three main Github repos
 
 | Event name and description                                          | Speaker      | Location | Date        |
 |---------------------------------------------------------------------|--------------|----------|-------------|
-| [Docker Belfast Meetup](https://www.meetup.com/Docker-Belfast/) | John McCabe | Belfast, Northern Ireland | 31-Oct-2017 |
```
Comment (alexellis) [inline community.md]: Looks like you deleted this bit?

**Comment by danigian** [comment]
Sorry for that @alexellis

**Comment by alexellis** [comment]
Looks like this removed some other changes so I've merged it on your behalf. Thank you for evaluating OpenFaaS - it'd be great to debrief with your team next week or individually.

**Pull request closed without merging**